src/jit/codegencommon.cpp

```diff
@@ -10317,10 +10333,23 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 
     compiler->unwindBegProlog();
 
+    inst_RV(INS_push, REG_FPBASE, TYP_REF);
+    compiler->unwindPush(REG_FPBASE);
+
     // TODO Save callee-saved registers
 
     // This is the end of the OS-reported prolog for purposes of unwinding
     compiler->unwindEndProlog();
+
+    // Restore frame pointer
+    // TODO We will need changes here when we introduce PSPSym
+    inst_RV_RV(INS_mov, REG_FPBASE, REG_ARG_0);
```

Comment:
@wateret That means that the unwinding in the CallEHFunclet works fine. So my guess is that there is a problem in the implementation of the ThrowExceptionFromContextInternal. It probably doesn't restore state for the exception throwing properly. As you can see from the failure stack, the next frame after CallEHFunclet is a garbage while it should be a frame of ExceptionTracker::CallHandler.
Could you please dump register values when you hit the breakpoint at the `call edx` and then after you hit the abort issue, dump register values at frame 8 (the one of the CallEHFunclet)? MY guess is that the esp or ebp would differ.